app/views/pulitzer/partials/_form.html.erb

```diff
@@ -34,4 +34,12 @@
       <%= ajax_delete 'delete', partial_path(partial), {class: 'button'}, dom_target(partial) %>
     </div>
   <% end %>
-</div>
\ No newline at end of file
+</div>
```

Comment:
@gomayonqui could you move this to the _show partial instead in the collapsed div?
https://github.com/edraut/pulitzer/blob/4414e035156714766fa00b54d25c8a034e1cf42e/app/views/pulitzer/partials/_show.html.erb#L22
Thanks!